backend/src/graphql.ts

```diff
@@ -55,10 +56,12 @@ schema.objectType({
     t.model.title()
     t.model.body()
     t.model.excerpt()
+    t.model.readTime()
     t.model.author()
     t.model.status()
     t.model.threads()
     t.model.language({ type: 'Language' })
+    t.string('createdAt')
```

Comment:
I couldn't figure out how to add `createdAt` as a regular `t.model.X` or "DateTime", nor could I find docs :( Right now this is working but I think preferably we'd send `createdAt` as a number / DateTime